test/orogen_test_helpers.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module IODriversBase
+    # Common helpers for the rather complex setup of iodrivers_base::Task tests
+    #
+    # It relies on the test cases to deploy a task-under-test and store it in the
+    # @task instance variable
+    module OroGenTestHelpers
+        # The task under test
+        attr_reader :task
```

Comment:
Please, let the user define his/her own readers.. I tend to use `let :task { ... }` a lot, which clashes with this